spec/unit/base/Version.spec.js

```diff
@@ -29,3 +32,30 @@ describe('fetch method', function() {
   });
 });
 
+describe('Paging limit', function () {
```

Comment:
We may want to have similar tests as these other fixes:
https://github.com/twilio/twilio-ruby/pull/530/files
https://github.com/twilio/twilio-python/pull/523
https://github.com/twilio/twilio-php/pull/654/files

Reply:
Thanks for the examples Elmer!